model/task/generate_tasks_estimations.go

```diff
@@ -0,0 +1,69 @@
+package task
+
+import (
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/utility"
+	adb "github.com/mongodb/anser/db"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+const (
+	numRevisionsToSearch = 10
```

Comment:
didnt see much difference for averages generated by 10 and 20 but we can change it